readthedocs/doc_builder/python_environments.py

```diff
@@ -276,9 +298,11 @@ def install_core_requirements(self):
             cwd=self.checkout_path  # noqa - no comma here in py27 :/
         )
 
-    def install_user_requirements(self):
-        requirements_file_path = self.config.python.requirements
-        if not requirements_file_path and requirements_file_path != '':
+    def install_requirements_file(self, install):
+        requirements_file_path = install.requirements
+        # This only happens when the config file is from v1.
```

Comment:
What is "This" here that happens on v1 only?

Reply:
the conditional below

Reply:
I think it's better to put the comment inside the `if` then.